lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -57,11 +58,38 @@ def self.run_dsym_upload(params)
             upload_url = dsym_upload_details['upload_url']
 
             UI.message("Uploading dSYM...")
-            Helper::AppcenterHelper.upload_dsym(api_token, owner_name, app_name, dsym_path, symbol_upload_id, upload_url)
+            Helper::AppcenterHelper.upload_symbol(api_token, owner_name, app_name, dsym_path, "Apple", symbol_upload_id, upload_url)
           end
         end
       end
 
+      def self.run_mapping_upload(params)
+        values = params.values
+        api_token = params[:api_token]
+        owner_name = params[:owner_name]
+        app_name = params[:app_name]
+        file = params[:apk]
```

Comment:
Do we need this?

Reply:
We don't need `file`, but all the rest is required